pyanaconda/anaconda.py

```diff
@@ -191,8 +191,11 @@ def dumpState(self):
         os.write(fd, dump_text_bytes)
         os.close(fd)
 
+        def opener(path, flags):
+            return os.open(path, flags, 0o600)
+
         # append to a given file
-        with open("/tmp/anaconda-tb-all.log", "a+") as f:
+        with open("/tmp/anaconda-tb-all.log", "a+", opener=opener) as f:
```

Comment:
Please replace the custom opener with the already existing helper function:

`from pyanaconda.core.path import open_with_perm`

Reply:
I've added changes. Please have a look.